sormas-backend/src/main/java/de/symeda/sormas/backend/sample/SampleService.java

```diff
@@ -181,11 +181,11 @@ public List<SampleIndexDto> getIndexList(SampleCriteria sampleCriteria, Integer
 		SampleJoins joins = sampleQueryContext.getJoins();
 
 		final Join<Sample, Case> caze = joins.getCaze();
-		final Join<Case, District> caseDistrict = joins.getCaseDistrict();
+		Join<Case, District> caseDistrict = caze.get(Case.DISTRICT) != null ? joins.getCaseDistrict() : joins.getCaseResponsibleDistrict();
```

Comment:
caze.get(Case.DISTRICT) != null  is always true.

BTW I re-checked the logic in this method and it would be ok to use the responsible district only

Reply:
changed back to initial approach